test/jdk/java/util/zip/ZipFile/InvalidBytesInEntryNameOrComment.java

```diff
@@ -84,24 +85,29 @@ public void setup() throws IOException {
      * the name field of a CEN file header should throw a
      * ZipException with "bad entry name or comment"
      */
-    @Test(expectedExceptions = ZipException.class,
-            expectedExceptionsMessageRegExp = BAD_ENTRY_NAME_OR_COMMENT)
+    @Test
     public void shouldRejectInvalidName() throws IOException {
-        try (ZipFile zf = new ZipFile(invalidName.toFile())) {
-            // Should throw ZipException
-        }
+        ZipException ex = expectThrows(ZipException.class, () -> {
+            openZipFile(invalidName);
```

Comment:
You could just do

```
ZipException ex = expectThrows(ZipException.class, () -> {
             new ZipFile(invalidName.toFile())
});
```
versus adding _openZipFile()_

Reply:
> The habit of opening resources in a TwR is hard to break, but I guess it's ok for a test like this. I have inlined the method and removed the TwR.

Agree, thanks for addressing the suggestion as it makes the test cleaner given expects/assertThrows should  react to the Exception being thrown by new ZipFile

Reply:
My inlining accidentally made the shouldRejectInvalidComment test the invalid name file. Fixed with last commit.